### DISCUSSION TITLE
"Daily" Standup - Nov 20, 2023

### DISCUSSION THREAD

**Discussion by warrenwallis** [warrenwallis/cs130-project#17] [General]
Team member's contributions towards project 😊

**Comment by JonahJKim** [comment]
did research towards GPT-connection and how to structure the backend infrastructure to scale.

**Comment by warrenwallis** [comment]
## Progress
- began initial development of frontend
  - created login page being able to input a username and password
  - created a home page with an input textbox and submit
- researched the best technologies to use for frontend and backend 

## Next Steps
- refactor page hierarchy to connect login to firebase and be able to log in with firebase authentication
  - reimplement login page
- implement all frontend parts needed for full application

**Comment by ArvinDing** [comment]
Will be working on the UI for the chat Page to allow people to see their messages and the messages from ChatGPT. Also reviewed a Warren's PR.
